includes/classes/db/mysql/query_factory.php

```diff
@@ -477,23 +477,25 @@ public function queryTime(): int
      * @param string $whereCondition condition for UPDATE (exclude the word "WHERE")
      * @param false $debug developer use only
      */
-    public function perform(string $tableName, array $tableData, $performType = 'INSERT', string $whereCondition = '', $debug = false): void
+    public function perform(string $tableName, array $tableData, string $performType = 'INSERT', string $whereCondition = '', ?bool $debug = false): void
     {
         switch (strtolower($performType)) {
+            case 'insertignore':
+                $insertString = "INSERT IGNORE INTO " . $tableName . " (";
             case 'insert':
-                $insertString = "INSERT INTO " . $tableName . " (";
+                $insertString = $insertString ?? "INSERT INTO " . $tableName . " (";
```

Comment:
For dis-ambiguity, should the `"INSERT INTO " . $tableName . " ("` portion be wrapped in parentheses?

Reply:
Done. Albeit differently.